doc/classes.md improvements (#610)
Akuli/jou: doc/classes.md

[padding](https://stackoverflow.com/questions/4306186/structure-padding-and-packing),
but this can be almost always ignored.

To create an instance of `Point`,

> we simply need enough memory to hold the two `int`s,
> and we need to tell the compiler to treat it as a `Point` instance.
> In other words, we simply create a variable whose type is `Point`:

Author: ```suggestion
we simply need to take enough memory to hold the two `int`s,
and we need to tell the compiler to treat that memory as a `Point` instance.
In other words, we create a variable whose type is `Point`:
```

    y: int

def main() -> int:
    p: Point
    p.x = 12
    p.y = 34
    printf("%d, %d\n", p.x, p.y)  # Output: 12, 34

>     p.y++

Author: ```suggestion

    p.y++
```

    printf("%d, %d\n", p.x, p.y)  # Output: 12, 35
    return 0
```

Alternatively, we could create the instance in one line with `p = Point{x = 12, y = 34}`.
This syntax is explained in detail [below](#instantiating).

## Pointers

Instances of classes are often passed around as pointers.
To understand why, let's try to make a function

> that increments the `x` coordinate of a `Point`:

Author: ```suggestion
that increments the `y` coordinate of a `Point`:
```

>  ```python

Author: ```suggestion
```python
```

    return 0
```

> The problem is that when we do `increment_y()`,
> we simply pass the 64 (or more) bytes of the struct to the `increment_y()` method.

Author: ```suggestion
The problem is that when we do `increment_y(p)`,
we simply pass the 64 (or more) bytes of the instance `p` to the `increment_y()` function.
```

> ## Instantiating

Author: ```suggestion
## Instantiating Syntax
```
